enterprise/backend/src/metabase_enterprise/content_translation/dictionary.clj

```diff
@@ -51,7 +52,12 @@
 (defn- format-row
   "Formats a row to be inserted into the content translation table. Locales are standardized, and all fields are trimmed. Extra fields are included as well."
   [headers row]
-  (let [{locale :language msgid :string msgstr :translation} (zipmap headers row)
+  (let [row-with-headers (zipmap headers row)
+        locale (or (get row-with-headers "language")
```

Comment:
To me it feels super odd to make our code looser to fit the copy instead of making the copy tighter to make it more clear exactly what we expect.

Instead of saying

```
The dictionary must be a CSV with these columns:
- Locale Code
- String
- Translation
```

Why not be very specific in our instructions and say

```
The dictionary must be a CSV with these columns:
- `language` (the locale code that should be used, e.g. `de` or `pt_BR`)
- `string` (the string to be translated)
- `translation` (the resulting translation for this language and string)

The column names are case-insensitive, so for example you may use `Language` or `LANGUAGE` instead of `language` if desired.
```